packages/eslint-plugin-pf-codemods/lib/rules/v5/td-TdSelectType-TdActionsType-rename-disable.js

```diff
@@ -0,0 +1,94 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8861
+// https://github.com/patternfly/patternfly-react/pull/8904
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context) {
+    const tableTdImport = getPackageImports(
+      context,
+      "@patternfly/react-table"
+    ).find((specifier) => "Td" === specifier.imported.name);
+
+    const tdName = tableTdImport?.local.name;
+
+    return !tableTdImport
+      ? {}
+      : {
+          JSXOpeningElement(node) {
+            if (tdName !== node.name?.name) {
+              return;
+            }
+
+            const attributes = node.attributes.filter(
```

Comment:
Nit: var name could be `attribuesToUpdate` or something like that, since we're only looking for specific attributes vs all of them